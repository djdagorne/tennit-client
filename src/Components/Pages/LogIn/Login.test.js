import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Login from './Login'
import '../../../setupTest'



it('renders without crashing', () => {
        const wrapper = shallow(<Login />)
    expect(toJson(wrapper)).toMatchSnapshot()
})
