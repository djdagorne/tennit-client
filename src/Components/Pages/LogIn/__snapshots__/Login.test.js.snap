// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="popup"
>
  <div
    className="popup-inner"
  >
    <button
      className="close-popup"
      onClick={[Function]}
    >
      X
    </button>
    <h2>
      Log In
    </h2>
    <form
      id="sign-up"
    >
      <div />
      <div
        className="form-item"
      >
        <label
          htmlFor="email"
        >
          Email (john@email.com)
        </label>
        <input
          name="email"
          placeholder="john@email.com"
          required={true}
          type="email"
        />
      </div>
      <div
        className="form-item"
      >
        <label
          htmlFor="Password"
        >
          Password (AAaa11!!)
        </label>
        <input
          name="password"
          placeholder="********"
          required={true}
          type="password"
        />
      </div>
      <div
        className="button-wrap"
      >
        <button
          className="rounded-button"
          type="submit"
        >
          Log in
        </button>
      </div>
    </form>
  </div>
</div>
`;
